### ISSUE TITLE
[New App Shortcut]: ZINEMA 5.0

### ISSUE THREAD

**Issue by drahgirl** [PRProd/HA-Firemote#515]
### App Name

Zinema

### Device Platform

Amazon Fire

### App Store Link and/or URL to .apk file if sideloaded

https://linktr.ee/underground_apps

### Required Information

"/system/bin/sh: adb: inaccessible or not found"

### Optional Comments

I actually have another app (HDO BOX) that I would love for you to add also! Really appreciate your build and I've been studying it for the last week and you're the only one that has delivered! THANK YOU THANK YOU THANK YOU for your amazing work on this!

**Comment by PRProd** [comment]
Greetings @drahgirl and welcome to GitHub!

I really appreciate your kind words!  I'm glad that you've found Firemote to be helpful.  I do have a few questions about your request(s).

## Zinema 5.0
Can you check again with the [procedure listed here](https://github.com/PRProd/HA-Firemote/wiki/Providing-Details-for-App-Launcher-Requests#for-fire-tv-chromecast-nvidia-shield-xiaomi-mi-and-other-android-based-devices) and/or the [video instructions here](https://youtu.be/XMBs51zkD7k) to see if you get a different result for the **Required Information** step?  The text you posted was actually an error message which was not what I was looking for.  Your result should have looked similar to this:
```
adb_response: mSurface=Surface(name=)/@0xe756dcd
      mSurface=Surface(name=com.cinema.hdintro/com.rtx.login.MOD.activity.login)/@0xef43353
  mCurrentFocus=Window{a6ee5e9 u0 com.cinema.hdintro/com.rtx.login.MOD.activity.login}
  mFocusedApp=AppWindowToken{e17356c token=Token{cb5de1f ActivityRecord{985bdbe u0 com.cinema.hdintro/com.rtx.login.MOD.activity.login t1101}}}
```
<br>

In addition to adding the launcher button for the **Device Platform** that was indicated in the request, I also will attempt to download the app on other devices to see if it is compatible there too.  If so, I will add the button for those other devices as well. [(see chart here)](https://github.com/PRProd/HA-Firemote/wiki/Firemote-App-Launchers).  When downloading this app from the location you provided, on the devices that had the Google Play Protect feature active, installation would be blocked with the following message:
![image](https://github.com/user-attachments/assets/a197beff-7fc5-4c22-9107-c659b93de1cb)
<br>

When I did successfully run this APK on a Fire device, I happened to find that the term 'MOD' was included in the Android Main Activity path.  While this does not mean anything by itself, I have seen this word attached to modified and/or malicious variants of valid Android apps, so that also raises concern for me.  

For those reasons, I brought [this .apk](https://underground.underground.appboxes.co/wl/?id=LLF575zCZrQsaKhsONMCbznI4JzCxFuH&fmode=download) over to [virustotal](https://www.virustotal.com) to have it scanned.  It was the first time this file had been scanned there (which might or might not be suspicious) but overall the scan shows that it seems to be OK. I will be adding this to the Firemote code if that is what you'd like me to do, but for Amazon Fire devices only.  [Click here to see the scan results](https://www.virustotal.com/gui/file/f11dce5b6febdecbda707caf681c8ce77948d3de27c74f354c99a8025cb93b26/detection)

For your own security and privacy, I would recommend that you think about and decide for yourself if you wish to continue using this app and/or if you still wish to have it included here.
<br>

## HDO Box
Normally I would request that you open a [new issue](https://github.com/PRProd/HA-Firemote/issues/new?assignees=PRProd&labels=New+App+Shortcut%2Cenhancement&projects=&template=App_Shortcut_Request.yaml&title=%5BNew+App+Shortcut%5D%3A+) for each app request, but I won't make you do that in this case.  In the future though, please make a separate request for each app launcher button.

Aside from that everything else looks good with the HDO Box request, so I'm working on it right now.
<br>

I look forward to hearing back from you!  Thanks!

-Doug

**Comment by PRProd** [comment]
@drahgirl ~

Just checking in.  It's been a week now, and I haven't heard back from you.  Please let me know about the Zinema 5.0 app.  After reading my comments above, I'd like to know if you still want me to proceed with adding it.

Thanks.

-Doug

**Comment by drahgirl** [comment]
﻿My apologies for the delay!I really appreciate all of this! I did verify that Zinema is good to go and safe on our end as well. Below are the adb_responses I got for both. Next time I will be sure to separate the tickets! ZINEMA 5.0 – adb_response: mAnimationIsEntrance=true ***@***.*** ***@***.*** mInputMethodTarget in display# 0 Window{9d9eb65 u0 com.cinema.hdintro/com.movie.ui.activity.shows.ShowActivity} HDO Box – adb_response: ***@***.*** mInputMethodTarget in display# 0 Window{18f7f33 u0 com.tv.hdobox/com.tv.hdobox.MainActivity} On Oct 7, 2024, at 3:21 PM, Doug Nelson ***@***.***> wrote:﻿
@drahgirl ~
Just checking in.  It's been a week now, and I haven't heard back from you.  Please let me know about the Zinema 5.0 app.  After reading my comments above, I'd like to know if you still want me to proceed with adding it.
Thanks.
-Doug

**Comment by PRProd** [comment]
Good News @drahgirl!

I have just published Firemote beta version [v4.1.0b2](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0b2) which contains both of your app shortcut requests.  Feel free to download and test this beta version, or you may also wait until it's officially released.  I will leave this issue open until these apps have been pushed into the official version.

Enjoy!

-Doug
<hr>

After updating to the newest version, if you are unable to see the new updates on your Firemote card, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) for help.

**Comment by PRProd** [comment]
@drahgirl

An update for you: I have just published Firemote version [v4.1.0](https://github.com/PRProd/HA-Firemote/releases/tag/v4.1.0) which contains your app shortcut request.

-Doug

**Comment by drahgirl** [comment]
Yay!! Thank you so so much! On Oct 18, 2024, at 12:23 AM, Doug Nelson ***@***.***> wrote:﻿
@drahgirl
An update for you: I have just published Firemote version v4.1.0 which contains your app shortcut request.
-Doug
